src/System.ServiceModel.Security/tests/ServiceModel/BinarySecretSecurityTokenTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System;
+using System.IdentityModel.Tokens;
+using System.ServiceModel.Security.Tokens;
+using Infrastructure.Common;
+using Xunit;
+
+public static class BinarySecretSecurityTokenTest
+{
+    [WcfFact]
+    public static void Ctor_Default_Properties()
+    {
+        string id = "test id";
+        byte[] keyBytes = new byte[128];
+        Random rnd = new Random();
+        rnd.NextBytes(keyBytes);
+
+        BinarySecretSecurityToken bsst = new BinarySecretSecurityToken(keyBytes);
+        Assert.NotNull(bsst);
+        Assert.NotNull(bsst.Id);
```

Comment:
When using this constructor, the id is auto-generated and is supposed to be unique. Can you add a test to validate that a second instance of BinarySecretSecurityToken using this constructor has a different Id?

Reply:
Thanks for review! Added a new test method for this.